site/ja/xla/tutorials/xla_compile.ipynb

```diff
@@ -0,0 +1,373 @@
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "kYpCXCdRHNuN",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "[y] = xla.compile(build_mnist_model, inputs=[images, labels])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "4giQh62IrZGF"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "グラフをコンパイルするとき、XLAはターゲット関数によって構築されたグラフのノードを、いくつかのXLAのオペレータで置き換えます。\n",
+        "\n",
+        "xla.compileは、生成されたXLAのオペレータから独立して実行できる `tf.Operation` を返しません\n",
+        "代わりに、ターゲット関数から返された `tf.Operation` ノードは、返された全ての `tf.Tensor` の値との制御依存関係として追加されます。これにより、 返されたテンソルが評価されるときに、 `tf.Operation` ノードの実行をトリガします。\n",
+        "\n",
+        " 擬似コードによるxla.compileの実装は、以下のようになります：\n",
+        "\n",
+        "---\n",
+        "```\n",
+        "# TensorFlowに、XLAが扱いやすい方法でコードを実行するよう依頼する\n",
+        "\n",
+        "y, train_step = build_mnist_model(images, labels)\n",
+        "with tf.control_dependencies([train_step]):\n",
+        "  y = tf.identity(y)\n",
+        "\n",
+        "# TensorFlowに、XLAが扱いやすい方法でコードの実行を停止するよう依頼する\n",
+        "```\n",
+        "---\n",
+        "\n",
+        "xla.compile()は常に `tf.Tensor` のリスト（１要素しか無かったとしても）を返します。"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "TPGas4jjFLZl"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "もしあなたが構築したグラフを今表示したら、通常のTensorFlowグラフとそれほど変わらないということを見て、前に述べたXLAのオペレータを見つける事ができません。これは、あなたが `sess.run()` でグラフを実行しようとしても、実際のコンパイルは後ほど発生するからです。後ほど、TensorFlowは実際にXLAオペレータを生成する一連のグラフ書き換えパスをトリガーします。これは、すべての入力がそろったときに、計算をコンパイルして実行します。"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "EZD1m_n1DxAF"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# モデルの学習とテスト"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "qe28bAHNHUG2",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# セッションを作成しすべての変数を初期化。\n",
+        "# xla.compile()は、Keras model.fit() APIやTF eager modeとはまだ 動作しません。\n",
+        "sess = tf.Session()\n",
+        "sess.run(tf.global_variables_initializer())"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "qgsKmz3n2UiW"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "以下のコードはモデルを学習します。 `y` の評価は、制御依存関係がある `train_step` をトリガします。これは、モデル変数を更新します。"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "_GxF6jTRHVuA",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        },
+        "outputId": "fbf299ca-02d5-4e95-f9fe-8f3c0432d132"
+      },
+      "cell_type": "code",
+      "source": [
+        "# 学習用データセットを与える\n",
```

Comment:
[Weak Suggestion]
In L147, `dataset` is being used instead of `データセット`.
I'm not sure which expression is better, but I think it is better to use same expression.

Reply:
Thanks. They should be consistent. I will use `データセット` at L147.